---
layout: page
title: bulk RNA-Seq
grand_parent: Modules
parent: Transcriptomics
has_children: false
nav_order: 1
---

# bulk RNA-Seq workshop
[GitHub Repository](https://github.com/hds-sandbox/bulk_RNAseq_course)

Updated: November 2021
Status: Under expansion

This workshop contains a basic tutorial on how to approach RNAseq experiments, starting from fastq files out of the sequencer. Thus, the workshop does not include any information about laboratory protocols, library preparation or any wet-lab related procedures. This workshop is based on a collection of modified tutorials from the [nf-core RNASeq](https://nf-co.re/rnaseq) pipeline, ....

## Syllabus:
1. Introduction to (tool) 
2. Preprocessing of (files)
	- Trimming (trim tool)
	- Alignment (tool)
3. Read norm (tool)
4. Exploratory (tool)
5. Differential (tool)
6. Functional (tool)

## Workshop requirements:
- Knowledge of R, Rstudio and Rmarkdown
- Basic knowledge of RNAseq tech...
- Basic knowledge of statistics such as PCA...

## Intended use
The aim of this repository is to run a comprehensive but introductory workshop on bulk-RNAseq bioinformatic analyses. Each of the modules of this workshop is accompanied by a powerpoint slideshow explaining...

## Acknowledgements:
- [Center for Health Data Science](https://heads.ku.dk/), University of Copenhagen.
- [Collaborator name](collaborator link), Bioinformatics Training Facility, University of Cambridge.
